mkdocs/docs/configuration.md

```diff
@@ -28,7 +28,7 @@ hide:
 
 There are three ways to pass in configuration:
 
-- Using the `~/.pyiceberg.yaml` configuration file
+- Using the `.pyiceberg.yaml` configuration file stored in either the directory specified by the `PYICEBERG_HOME` environment variable, the home directory, or current working directory.
```

Comment:
nit: move the extra info about where the file is located down to L37. 
also i think its valuable to include a warning about accidentally checking in secrets with git when using the current working directory